concepticondata/conceptlists/Gabelentz-1892-1074.tsv

```diff
@@ -0,0 +1,1075 @@
+Gabelentz-1892-1074-771	771	Ob	if	VI. Conjunctionen			
+Gabelentz-1892-1074-772	772	Obschon	although	VI. Conjunctionen			
+Gabelentz-1892-1074-773	773	Zu (Richtung wohin)	to (direction)	VII. Präpositionen		2754	TOWARDS
+Gabelentz-1892-1074-774	774	Dativ	(realization of dative case)	VII. Präpositionen			
+Gabelentz-1892-1074-775	775	Mit (in Begleitung von…)	with	VII. Präpositionen		1340	WITH
+Gabelentz-1892-1074-776	776	vermittels	by means of	VII. Präpositionen			
+Gabelentz-1892-1074-777	777	Durch	through	VII. Präpositionen		97	THROUGH
+Gabelentz-1892-1074-778	778	Gegen	against	VII. Präpositionen		2469	AGAINST
+Gabelentz-1892-1074-779	779	Wegen	because	VII. Präpositionen		1157	BECAUSE
+Gabelentz-1892-1074-780	780	Um (etwas herum)	around something	VII. Präpositionen			
+Gabelentz-1892-1074-781	781	Von (etwas her)	from (somewhere)	VII. Präpositionen		693	FROM
+Gabelentz-1892-1074-782	782	Genitiv	(realization of genitive case)	VII. Präpositionen			
+Gabelentz-1892-1074-783	783	In (etwas drinnen)	inside	VII. Präpositionen		1460	IN
+Gabelentz-1892-1074-784	784	in (etwas hinein)	inside (movement)	VII. Präpositionen			
+Gabelentz-1892-1074-785	785	Für	for	VII. Präpositionen			
+Gabelentz-1892-1074-786	786	Bei (dicht bei, nahe bei)	nearby	VII. Präpositionen			
+Gabelentz-1892-1074-787	787	An (der Seite von…)	at (someone’s side)	VII. Präpositionen		1461	AT
+Gabelentz-1892-1074-788	788	Sprechen, reden, sagen	to speak	VIII. Verba	A. Sagen, sprechen u.s.w.	1623	SPEAK
+Gabelentz-1892-1074-789	789	Schreien	to shout, to scream	VIII. Verba	A. Sagen, sprechen u.s.w.	715	SHOUT
+Gabelentz-1892-1074-790	790	Heulen	to cry	VIII. Verba	A. Sagen, sprechen u.s.w.	1839	CRY
+Gabelentz-1892-1074-791	791	Rufen	to call	VIII. Verba	A. Sagen, sprechen u.s.w.	1084	CALL
+Gabelentz-1892-1074-792	792	Flüstern	to whisper	VIII. Verba	A. Sagen, sprechen u.s.w.	55	WHISPER
+Gabelentz-1892-1074-793	793	Flöten, pfeifen	to whistle	VIII. Verba	A. Sagen, sprechen u.s.w.	1025	WHISTLE
+Gabelentz-1892-1074-794	794	Singen	to sing	VIII. Verba	A. Sagen, sprechen u.s.w.	1261	SING
+Gabelentz-1892-1074-795	795	Schnalzen	to click one’s tongue	VIII. Verba	A. Sagen, sprechen u.s.w.		
+Gabelentz-1892-1074-796	796	Fragen	to ask	VIII. Verba	A. Sagen, sprechen u.s.w.	1309	ASK (INQUIRE)
+Gabelentz-1892-1074-797	797	Antworten	to answer	VIII. Verba	A. Sagen, sprechen u.s.w.	1275	ANSWER
+Gabelentz-1892-1074-798	798	Winken	to wave	VIII. Verba	A. Sagen, sprechen u.s.w.	3544	WAVE (VERB)
+Gabelentz-1892-1074-799	799	Beten	to pray	VIII. Verba	A. Sagen, sprechen u.s.w.	24	PRAY
+Gabelentz-1892-1074-800	800	Bitten	to ask for	VIII. Verba	A. Sagen, sprechen u.s.w.	1828	ASK (REQUEST)
+Gabelentz-1892-1074-801	801	Danken	to thank	VIII. Verba	A. Sagen, sprechen u.s.w.	2412	THANK (SOMBEODY)
+Gabelentz-1892-1074-802	802	Loben	to praise	VIII. Verba	A. Sagen, sprechen u.s.w.	3455	COMMEND
+Gabelentz-1892-1074-803	803	Fluchen	to swear	VIII. Verba	A. Sagen, sprechen u.s.w.	1939	SWEAR
```

Comment:
Is 'fluchen' here similar to 'schimpfen'? Because if it has a similar meaning, then it can't be mapped to SWEAR which in conception means 'to make an oath'.

Reply:
Yes, it's usually to say swear words (i.e. like schimpfen). So yes, not SWEAR.

Reply:
You are completely right, I missed the concept definition.